src/metabase/email/messages.clj

```diff
@@ -327,24 +328,26 @@
   []
   (or (driver/report-timezone) "UTC"))
 
-(defn alert-schedule-text
-  "Returns a string that describes the run schedule of an alert (i.e. how often results are checked),
-  for inclusion in the email template. Not translated, since emails in general are not currently translated."
-  [channel]
-  (case (keyword (:schedule_type channel))
-    :hourly
-    "Run hourly"
-
-    :daily
-    (format "Run daily at %s %s"
-            (schedule-hour-text channel)
-            (schedule-timezone))
-
-    :weekly
-    (format "Run weekly on %s at %s %s"
-            (schedule-day-text channel)
-            (schedule-hour-text channel)
-            (schedule-timezone))))
+(defn notification-card-schedule-text
```

Comment:
shouldn't this be a helper for Handlebars?

Reply:
yep it should, I plan to do another round of removing these computed stuffs soon